### ISSUE TITLE
Mention broken status on VS Marketplace

### ISSUE THREAD

**Issue by laggingreflex** [pzuraq/vscode-vertical-tab-bar#6]
Sad to see this no longer works. I came across this from some old thread. 

Probably worth mentioning on the marketplace that this doesn't work: https://marketplace.visualstudio.com/items?itemName=pzuraq.vertical-tab-bar

**Comment by pzuraq** [comment]
Yeah I would unlist this, but I'm not sure that's possible? I just haven't had time to maintain unfortunately. Still planning on trying to update it when I get a chance, it was quite useful when you have a ton of files open.
